src/pybind/mgr/cephadm/module.py

```diff
@@ -1318,6 +1313,11 @@ def _check_valid_addr(self, host: str, addr: str) -> str:
 > ssh -F ssh_config -i ~/cephadm_private_key {self.ssh_user}@{addr}'''
             raise OrchestratorError(msg)
 
+        if not addr or host == addr:  # In case we had to deduce the IP ourselves
```

Comment:
ok this is not going to work. My use case it to quickly have a local single-host ceph cluster with 127.0.0.1 as this is my typical testing setup. Any ideas?

Reply:
maybe `addr == ip_addr` might work.

Reply:
i think the way the current code work you can bootstrap a cluster with --mon-ip 127.0.0.1 and additionally you can add a host with 127.0.0.1 as the address. the current code just prevent the mgr from try to resolve its own address which would result in 127.0.0.1   but if you really want to have 127.0.0.1 as the address its possible